app/representers/api/musical_work_representer.rb

```diff
@@ -9,7 +9,7 @@ class Api::MusicalWorkRepresenter < Api::BaseRepresenter
   property :label
   property :album
   property :year
-  property :excerpt_length, as: :length
+  property :excerpt_length, as: :duration
```

Comment:
In `audio_version.rb` and `story.rb` models, the duration is defined at the model level, via alias or method.
I think for consistency it would be appropriate to alias `excerpt_length` to `duration` in the `musical_work.rb` model.